taxonomies/product/labels.txt

```diff
@@ -18,6 +19,18 @@ fr: Conforme RoHS
 en: WEEE
 fr: DEEE
 
+xx: UKCA
+en: UKCA marking, UK Conformity Assessed
```

Comment:
When we display the logo in English, en: will take priority, and we will display "UKCA marking".
It may be better to have:

```suggestion
en: UKCA, UKCA marking, UK Conformity Assessed
```

Reply:
Went ahead and changed CE as well as FCC and RoHS too.